### DISCUSSION TITLE
MediaPlayerElement is not available in XAML code for UNO

### DISCUSSION THREAD

**Discussion by apovetkin** [unoplatform/uno#9318] [General]
Hi guys,

Please help me to figure out why MediaPlayerElement is not available in xaml. 
Here I can see this element should be accessible https://platform.uno/docs/articles/controls/MediaPlayerElement.html

I installed latest version of UNO followed all instructions from here https://platform.uno/docs/articles/get-started-vs-2022.html?tabs=windows

Maybe somebody can share the simple solution of UNO with MediaPlayerElement or VLC for Visual Studio 2022?

Many thanks in advanced!!!

Alexander

**Comment by Daoting** [comment]
Because WinUI3 does not support `MediaPlayerElement` now. Works fine on UWP.
